app/views/security_group/show.html.haml

```diff
@@ -1,5 +1,5 @@
 #main_div
   - if ["instances"].include?(@display) && @showtype != "compare"
-    = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@security_group.id}"}
+    = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
```

Comment:
can't be done, can't use url_for and not specify the controller, so deeper refactoring will be needed